### PR TITLE
Add parser for spec to get the count of non-yum type repos

### PR DESCRIPTION
**Pull request by huali027**
Signed-off-by: Huanhuan Li <huali@redhat.com>

### CODE REVIEW

File: insights/specs/insights_archive.py [hunk 1]
```diff
@@ -210,6 +210,7 @@ class InsightsArchiveSpecs(Specs):
     saphostexec_status = simple_file("insights_commands/usr.sap.hostctrl.exe.saphostexec_-status")
     saphostexec_version = simple_file("insights_commands/usr.sap.hostctrl.exe.saphostexec_-version")
     satellite_mongodb_storage_engine = simple_file("insights_commands/mongo_pulp_database_--eval_db.serverStatus_.storageEngine")
+    satellite_non_yum_type_repos = simple_file("insights_commands/mongo_pulp_database_--eval_db.repo_importers.find_importer_type_id_ne_yum_importer_.count")
```
Comment (xiangce) [inline insights/specs/insights_archive.py:213]: This is not necessary for specs that depend on other @datasource.

Comment (huali027) [reply insights/specs/insights_archive.py:213]: @xiangce, sure, I'll remove it, thanks.

File: insights/specs/default.py [hunk 2]
```diff
@@ -678,11 +678,24 @@ def is_satellite_server(broker):
             return True
         raise SkipComponent
 
+    @datasource(CapsuleVersion)
```
Comment (bfahr) [inline insights/specs/default.py:681]: @huali027 please change this decorator to include `HostContext` like this: `@datasource(CapsuleVersion, HostContext)`.  See #2908 and #2911 for more information.

**Review by bfahr — CHANGES_REQUESTED** [review]

File: insights/specs/default.py (hunk 2 above)
Comment (huali027) [reply insights/specs/default.py:681]: @bfahr, sure, I'll update it.

**Review by xiangce — APPROVED** [review]
Looks good to me, pending the approval of spec.

**Review by bfahr — APPROVED** [review]

**Pull request merged**